Src/AutoFixtureUnitTest/Kernel/TerminatingWithPathSpecimenBuilderTest.cs

```diff
@@ -162,35 +159,62 @@ public void SpecimenRequestsWillInitiallyBeEmpty()
         [Fact]
         public void SpecimenRequestsRaisedFromTracerAreRecordedCorrectly()
         {
-            // Fixture setup
-            var tracer = new DelegatingTracingBuilder();
-            var specimens = new[] { new object(), new object(), new object() };
-            var requestEvents = specimens.Select((o, i) => new RequestTraceEventArgs(o, i)).ToList();
-            var sut = new TerminatingWithPathSpecimenBuilder(tracer);
-            // Exercise system
-            requestEvents.ForEach(tracer.RaiseSpecimenRequested);
-            // Verify outcome
-            Assert.True(specimens.SequenceEqual(sut.SpecimenRequests));
-            // Teardown
+            // arrange
```

Comment:
I used that AAA style as it's simpler as less verbose, so it was more convenient for me to write this. I didn't think it does matter, as `//Teardown` is mostly empty. 

However, if you feel this like a project conventions violation, I'll gladly fix that and will follow this convention in future. :wink: